app/src/main/java/com/example/bigowlapp/activityPage/SignUpPageActivity.java

```diff
@@ -86,11 +90,16 @@ public void onComplete(@NonNull Task<AuthResult> task)
                                 //if successful sign up
                                 else
                                 {
+                                    FirebaseUser currentUser = FirebaseAuth.getInstance().getCurrentUser();
                                     user.setEmail(email);
                                     user.setPhoneNumber(userPhone);
                                     user.setFirstName(userName);
-                                    //user.setUId();
-                                    startActivity(new Intent(SignUpPageActivity.this, HomePageActivity.class));
+                                    user.setUId(currentUser.getUid());
+
+                                    if(currentUser != null){
```

Comment:
I know this seems petty, but there should be a space on the if statement, like
`if(condition) {`


To make life easy, just run the command `CTRL+ALT+SHIFT+L` on Android Studio on this file and make it run on `Whole File` and do `Code Cleanup` (and you can make it do `Optimize imports` as well)

Reply:
I will do that to every single files for a starter.

Reply:
Thanks for doing this btw. It made a bunch of our code follow a unified style!